src/main/java/seedu/address/logic/commands/AddNoteCommand.java

```diff
@@ -18,32 +18,33 @@
 /**
  * Adds a person to the address book.
  */
-public class AddNoteCommand extends Command {
+public class AddNoteCommand extends UndoableCommand {
 
     public static final String COMMAND_WORD = "add-an";
     public static final Prefix PREFIX_DATE = new Prefix("d/");
     public static final Prefix PREFIX_TIME = new Prefix("t/");
     public static final Prefix PREFIX_NOTE = new Prefix("n/");
 
-
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds an appointment note to the specified person. "
-        + "Parameters: PATIENT_INDEX (must be a positive integer)"
-        + PREFIX_DATE + "DATE<DD-MM-YYYY> "
-        + PREFIX_TIME + "TIME<HHMM> "
-        + PREFIX_NOTE + "NOTE "
-        + "Example: " + COMMAND_WORD + " 1 "
-        + PREFIX_DATE + "19-02-2024 "
-        + PREFIX_TIME + "1430 "
-        + PREFIX_NOTE + "General Flu ";
+            + "Parameters: PATIENT_INDEX (must be a positive integer)"
+            + PREFIX_DATE + "DATE<DD-MM-YYYY> "
+            + PREFIX_TIME + "TIME<HHMM> "
+            + PREFIX_NOTE + "NOTE "
+            + "Example: " + COMMAND_WORD + " 1 "
+            + PREFIX_DATE + "19-02-2024 "
+            + PREFIX_TIME + "1430 "
+            + PREFIX_NOTE + "General Flu ";
 
     public static final String MESSAGE_SUCCESS = "New appointment note added: %1$s";
+    public static final String MESSAGE_UNDO_ADD_SUCCESS = "Add patient medical record undone.";
```

Comment:
Think it should be `Add appointment note undone.`

Reply:
Hi Sebin, thanks for catching the typo!